src/main/java/wolox/training/controllers/UserController.java

```diff
@@ -0,0 +1,91 @@
+package wolox.training.controllers;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.server.ResponseStatusException;
+import wolox.training.models.Book;
+import wolox.training.models.User;
+import wolox.training.repositories.UserRepository;
+
+@RestController
+@RequestMapping("/users")
+public class UserController {
+
+  public static final String USER_ID_DOES_NOT_MATCH_WITH_ID_PROVIDED = "User id (%d) does not match with id provided (%d).";
+  public static final String USER_WAS_NOT_FOUND_ID = "User was not found. ID: %d";
+  public static final String USER_WAS_NOT_FOUND_USERNAME = "User was not found. Username: %s";
```

Comment:
We usually put constants in a separated class (e.g. UserControllerConstants.java ) contained in a package named `constants`. But maybe @facucompe could tell you how they manage this things in your future project 😃

Reply:
Hi! I just put those constants in a ExceptionConstants class in feature/setters-preconditions ( just to avoid all those merge conflicts with the branches 😅  )